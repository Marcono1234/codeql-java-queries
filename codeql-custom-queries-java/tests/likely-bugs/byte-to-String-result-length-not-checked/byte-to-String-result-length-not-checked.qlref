./likely-bugs/byte-to-String-result-length-not-checked.ql
