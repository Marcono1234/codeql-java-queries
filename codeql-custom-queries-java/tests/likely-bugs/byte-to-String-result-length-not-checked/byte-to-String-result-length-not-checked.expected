| Test.java:4:19:4:48 | toString(...) | Does not check length of String representation |
| Test.java:5:19:5:47 | toString(...) | Does not check length of String representation |
| Test.java:6:19:6:46 | toString(...) | Does not check length of String representation |
| Test.java:7:19:7:50 | toBinaryString(...) | Does not check length of String representation |
| Test.java:8:19:8:49 | toOctalString(...) | Does not check length of String representation |
| Test.java:9:19:9:47 | toHexString(...) | Does not check length of String representation |
| Test.java:11:20:11:48 | toHexString(...) | Does not check length of String representation |
| Test.java:18:19:18:40 | format(...) | Should specify width and padding ("%02x") |
| Test.java:19:19:19:40 | format(...) | Should specify width and padding ("%02x") |
| Test.java:21:13:21:34 | format(...) | Should specify width and padding ("%02x") |
| Test.java:28:19:28:52 | toHexString(...) | Does not check length of String representation |
| Test.java:29:19:29:45 | format(...) | Should specify width and padding ("%02x") |
